src/pages/interim/_everything.js

```diff
@@ -17,14 +17,21 @@ const Ul = styled(Flex).attrs({ as: 'ul' })`
     flex-wrap: wrap;
 `
 const Li = styled(Flex).attrs({ as: 'li' })`
+    justify-content: left;
     max-width: 48.6rem;
     display: flex;
     margin: 2rem 1.2rem;
+    min-height: 120px;
 
     @media ${device.mobileL} {
         flex-direction: column;
     }
+
+    @media (max-width: 1214px) {
```

Comment:
device.laptopM?

Reply:
I use the custom max width because the layout changed exacty at 1214 px (device.laptopM is 1200 px)